README.md

```diff
@@ -27,7 +27,8 @@ Want to know more about Emission? Read the [mobile](http://artsy.github.io/blog/
    * `$ brew link pcre`
    * `$ brew install watchman --HEAD`
 3. Install NPM modules: `$ yarn install`
-4. Install Pods: `$ cd Example && pod install`
+4. Install Cocoapods: `$ gem install cocoapods`
+5. Install Pods: `$ cd Example && ARTSY_STAFF_MEMBER=true pod install`
```

Comment:
I was getting a crash on a fonts pod until I added this ENV var (and `pod install` prompted me to login as well).